### PR TITLE
Refs task #16569 Create Home UI

### PR DESCRIPTION
**Pull request by datdt-1935**
## Ticket title
- Refs task #16569 (Feature #16568 Create Home Screen)
## Redmine id
- [Task #16569](https://edu-redmine.sun-asterisk.vn/issues/16569)
## Screenshot or screencast
Home

### CODE REVIEW

File: SoundGood/Scenes/Home/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @IBOutlet weak var segmentController: UISegmentedControl!
+    @IBOutlet weak var trackTableView: UITableView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+    }
+
+    func setupTableView() {
```
Comment (hungdt-1518) [inline SoundGood/Scenes/Home/HomeViewController.swift]: private e nhé

File: SoundGood/Scenes/Home/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        trackTableView.register(cellType: TrackTableViewCell.self)
+    }
+
+    @IBAction func segmentActionChanged(_ sender: Any) {
+        trackTableView.reloadData()
+    }
+}
+
+extension HomeViewController: UITableViewDataSource {
+    // MARK: - Table view data source
```
Comment (hungdt-1518) [inline SoundGood/Scenes/Home/HomeViewController.swift]: đặt lên trước dòng extension

File: SoundGood/Scenes/Home/TrackTableViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+//  SoundGood
+//
+//  Created by Dang Thanh Dat on 8/23/19.
+//  Copyright © 2019 Sun Asterisk. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+class TrackTableViewCell: UITableViewCell, NibReusable {
```
Comment (longvt-1468) [inline SoundGood/Scenes/Home/TrackTableViewCell.swift]: thêm final nhé

File: SoundGood/Scenes/Home/TrackTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    @IBOutlet weak var trackImage: UIImageView!
```
Comment (longvt-1468) [inline SoundGood/Scenes/Home/TrackTableViewCell.swift]: thêm private các outlet nhé

**Review by tgdong2296 — APPROVED** [review]

**Review by hungdt-1518 — APPROVED** [review]

**Review by longvt-1468 — APPROVED** [review]

**Pull request merged**